app/jobs/catalogue_check_job.rb

```diff
@@ -0,0 +1,76 @@
+class CatalogueCheckWorker < ActiveJob::Base
+  queue_as :default
+
+  require 'capybara'
+  require 'capybara/poltergeist'
+  require_relative '../../app/jobs/scrape_job.rb'
+  require_relative '../../app/jobs/delete_job.rb'
+
+  include Capybara::DSL
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, timeout: 60, js_errors: false)
+  end
+  Capybara.default_driver = :poltergeist
+
+  def perform
+
+    #setting the initial variable
+    current_catalogue = Hash.new(0)

```

Comment:
I used Hash.new(0), because I came across this post: http://stackoverflow.com/questions/2698460/strange-ruby-behavior-when-using-hash-new
So i thought that it'll be safer to use this to avoid unwanted behavior.

any thought on better way to do this?